### PR TITLE
n28: shroom kingdom

### PR DESCRIPTION
**Pull request by Tarnadas**
_Part of #852_

The linkdrop campaign will be online very soon, so don't worry about the dead link.
I'll probably have something online later today.

### CODE REVIEW

File: content/news/028/index.md [hunk 1]
```diff
@@ -84,6 +84,44 @@ out [this form][gamedev-meetup-form].
...
+Currently a prototype of the game is in development.
+An asset file, that has been extracted from Super Mario Maker 2 game files with an
+[external program](asset-extractor), is loaded inside a browser to start the game.
+The game is written with the [Bevy game engine](bevy)
+and the [Rapier physics engine](rapier) and is compiled to WebAssembly.
+The prototype is still very basic, but you can already place and remove blocks
+and have a feeling about the physics recreation of the original games.
+
+In the past months there also has been the SHRM token launch and a
+[token airdrop](shrm-airdrop) for NEAR early adopters. The [Shroom Kingdom DAO](shrm-dao)
```
Comment (AngelOnFira) [inline content/news/028/index.md:105]: Links need to be square brackets, not circular ones

File: content/news/028/index.md [hunk 2]
```diff
@@ -84,6 +84,44 @@ out [this form][gamedev-meetup-form].
...
+![Shroom Kingdom game preview](shroom-kingdom.gif)
+_Starting the game and placing some blocks_
+
+Shroom Kingdom ([GitHub][shrm-github], [Discord][shrm-discord],
+[Twitter][shrm-twitter]) is an upcoming play-to-earn video game built with web
+technologies running on the [NEAR Blockchain][near-blockchain].
+
+Currently a prototype of the game is in development.
+An asset file, that has been extracted from Super Mario Maker 2 game files with an
+[external program](asset-extractor), is loaded inside a browser to start the game.
```
Comment (AngelOnFira) [inline content/news/028/index.md:98]: I'm curious, are there any copyright issues with the extraction?

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
I'm curious about the potential copyright issue. I think it would be best not to show a project that is ripping assets from somewhere else if that's the case here (I might not be understanding it properly, feel free to correct me :)

File: content/news/028/index.md (hunk 1 above)
Comment (Tarnadas) [reply content/news/028/index.md:105]: which link do you mean? I think I made them all in square brackets here

File: content/news/028/index.md (hunk 2 above)
Comment (Tarnadas) [reply content/news/028/index.md:98]: The asset extractor is using an open source tool for decryption and unpacking: https://github.com/SciresM/hactool
This tool can extract most of the game files to a container format, which can then be parsed with e.g. this: https://github.com/tarnadas/ninres-rs
And then I just convert the images to png.

The user needs to provide the console keys and the game files for decryption, so it should be fine.
SciresM is also the developer of atmosphere, a CFW for Nintendo Switch and hactool is probably used a lot as a library in several applications.

The container formats that I parse also have various different libraries in different languages, e.g. this one: https://github.com/KillzXGaming/Switch-Toolbox

No copyrighted assets are bundled in the app nor do we share any of them

File: content/news/028/index.md (hunk 1 above)
Comment (AngelOnFira) [reply content/news/028/index.md:105]: [Rapier physics engine](rapier) should be [Rapier physics engine][rapier]

Comment (Tarnadas) [reply content/news/028/index.md]: Oh yeah sorry. I was so used to writing links with regular brackets

**Comment by AngelOnFira** [comment]
Alright, so as per our discussions, the current path forward will be to remove the Gif and the section on extracting the copyright material.

Since I haven't been able to make these commits myself to the branch, I'll have to ask you to make them. I'm going to move forward with the draft release as it shouldn't wait any longer. You mentioned that I could just branch and commit that, but I don't want to go further out of our standard workflow.

If you get this done in the next few hours then I'll merge it in!

**Comment by AngelOnFira** [comment]
@Tarnadas it will also need to go at the bottom of the `### Games Update` section, can you move it down?

**Comment by Tarnadas** [comment]
rebased and moved down

**Review by AngelOnFira — APPROVED** [review]

**Pull request merged**